app/api/controllers/validators/validations/NinoValidation.scala

```diff
@@ -14,18 +14,20 @@
  * limitations under the License.
  */
 
-package api.controllers.requestParsers.validators.validations
+package api.controllers.validators.validations
 
 import api.models.domain.Nino
 import api.models.errors.{ MtdError, NinoFormatError }
 
-object NinoValidation {
+object NinoValidation extends Validation {
```

Comment:
For this class, did you think about making this a parser validation, so it returned `Nino(nino)` instead of `NoValidationErrors` in the success branch. It would save us having to do it separately in the endpoint validator.

Outside the scope of this ticket, but we noticed there are two separate nino regexes, one here and one called in the `Nino.isValid` function, that perhaps could be consolidated in the future?

Reply:
I've deliberately tried not to change the Validations themselves too much; bit of a rabbit-warren of work to discover there, and kind-of outside the scope of this refactoring. There's value in exploring that though; could  see a tech-debt ticket being added for this.